dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -706,3 +747,37 @@ fn json_to_hashmap(json: &str) -> Result<HashMap<String, String>, DscError> {
     }
     Ok(map)
 }
+
+/// Log output from a process as traces.
+///
+/// # Arguments
+///
+/// * `process_name` - The name of the process
+/// * `process_id` - The ID of the process
+/// * `stderr` - The stderr output from the process
+pub fn log_resource_traces(process_name: &str, process_id: &u32, stderr: &str)
+{
+    if !stderr.is_empty()
+    {
+        for trace_line in stderr.lines() {
+            if let Result::Ok(json_obj) = serde_json::from_str::<Value>(trace_line) {
+                if let Some(msg) = json_obj.get("Error") {
+                    error!("Process '{process_name}' id {process_id} : {}", msg.as_str().unwrap_or_default());
```

Comment:
It would be good to make `"Process '{process_name}' id {process_id}` part shorter; with it trace lines are too long.
Maybe `[<processname>:<procId>]: <text>`

Reply:
We can do that in a follow-up PR along with handling the `tracing` crate JSON format